EnglishTranslation-master/SKILL.tsv

```diff
@@ -9505,7 +9505,7 @@ SKILL_20170726_009504	Attack #{SkillFactor}#%{nl}No. of targets #{CaptionRatio}#
 SKILL_20170726_009505	{#DD5500}{ol}[Missile: Gun]{/}{/}{nl}Repeatedly fires at the enemy.	{#DD5500}{ol}[미사일:총]{/}{/}{nl}적에게 연속 사격하여 피해를 줍니다.	SKILL_20170510_009401	{#DD5500}{ol}[Missile: Gun]{/}{/}{nl}Repeatedly fire at one enemy.	{#DD5500}{ol}[미사일:총]{/}{/}{nl}적 한명에게 연속 사격하여 피해를 줍니다.	
 SKILL_20170726_009506	STR, INT: +#{CaptionRatio}#{nl}Effect Duration: #{CaptionRatio3}# seconds{nl}Stigma Duration: #{CaptionRatio2}# seconds{nl}No. of targets 5	힘, 지능 #{CaptionRatio}# 증가{nl}효과 지속시간 #{CaptionRatio3}#초{nl}낙인 지속시간 #{CaptionRatio2}#초{nl}낙인 적용 대상 수 5	SKILL_20170510_009416	STR, INT: +#{CaptionRatio}#{nl}Effect Duration: #{CaptionRatio3}# seconds{nl}Stigma Duration: #{CaptionRatio2}# seconds	힘, 지능 #{CaptionRatio}# 증가{nl}효과 지속시간 #{CaptionRatio3}#초{nl}낙인 지속시간 #{CaptionRatio2}#초	
 SKILL_20170726_009507	Attack: #{SkillFactor}#%{nl}Casting Time: #{CaptionTime}# seconds{nl}HP recovery after resurrection: #{CaptionRatio}#%{nl}No. of targets #{CaptionRatio2}#	공격력 #{SkillFactor}#%{nl}캐스팅 시간 #{CaptionTime}#초{nl}전투불능 회복 후 HP #{CaptionRatio}#% 회복{nl}부활 대상 수 #{CaptionRatio2}#	SKILL_20170420_008631	Attack: #{SkillFactor}#%{nl}Casting Time: #{CaptionTime}# seconds{nl}HP recovery after resurrection: #{CaptionRatio}#%	공격력 #{SkillFactor}#%{nl}캐스팅 시간 #{CaptionTime}#초{nl}전투불능 회복 후 HP #{CaptionRatio}#% 회복	
-SKILL_20170726_009508	Magic circle duration: #{CaptionTime}# sec{nl}Max. 8 zombies summoned{nl}Zombie damage increases with level.	마법진 #{CaptionTime}#초 지속{nl}최대 좀비 8마리 소환{nl}레벨 상승에 따라 좀비 대미지가 상승합니다.	SKILL_20150317_001484	Magic Circle Duration: #{CaptionTime}# {nl}Maximum Zombie Summons: #{CaptionRatio}#	마법진 #{CaptionTime}#초 지속{nl}최대 좀비 #{CaptionRatio}#마리 소환	
+SKILL_20170726_009508	Magic Circle Duration: #{CaptionTime}# sec{nl}Max. 8 zombies summoned{nl}Zombie damage increases with level.	마법진 #{CaptionTime}#초 지속{nl}최대 좀비 8마리 소환{nl}레벨 상승에 따라 좀비 대미지가 상승합니다.	SKILL_20150317_001484	Magic Circle Duration: #{CaptionTime}# {nl}Maximum Zombie Summons: #{CaptionRatio}#	마법진 #{CaptionTime}#초 지속{nl}최대 좀비 #{CaptionRatio}#마리 소환	
```

Comment:
`sec` -> `seconds` for consistency
`Max. 8 zombies summoned` - Could be phrased better. `Can summon up to 8 Zombies` ?
`Zombie damage increases with level.` - `Increases Zombie damage per level` ?